packages/webapp/src/containers/Help/saga.js

```diff
@@ -31,12 +32,26 @@ export function* supportFileUploadSaga({ payload: { file, form } }) {
     const formData = new FormData();
     formData.append('_file_', file);
     formData.append('data', JSON.stringify(form));
-    const result = yield call(axios.post, supportUrl(), formData, {
-      headers: {
-        'Content-Type': 'multipart/form-data',
-        Authorization: 'Bearer ' + localStorage.getItem('id_token'),
-      },
-    });
+    let result;
+    if (form.support_type === 'Delete my farm') {
+      const { farmUrl } = apiConfig;
+      let { user_id, farm_id } = yield select(userFarmSelector);
+      result = yield call(axios.delete, farmUrl + '/' + farm_id, {
```

Comment:
This feels like it should be its own isolated function, but I did it inline here for convenience to show something working. I don't have very much experience with generator functions